lib/middleware/locals.coffee

```diff
@@ -38,5 +39,14 @@ module.exports = (req, res, next) ->
   res.locals.sd.REFLECTION = req.headers?['user-agent']?.match('PhantomJS')?
   res.locals.sd.REQUEST_TIMESTAMP = Date.now()
   res.locals.sd.NOTIFICATION_COUNT = req.cookies?['notification-count']
+  res.locals.sd.USER_AGENT = res.locals.userAgent = req.get('user-agent')
+  ua = req.get('user-agent')
+  res.locals.sd.IS_MOBILE = Boolean(
```

Comment:
This `locals` middleware is merged from desktop/mobile, and this line where we set `sd.IS_MOBILE` is where we determine [whether to route to the mobile app or desktop here](https://github.com/artsy/force/pull/1197/files#diff-f5d8c0f33aa88aa6e119a0cdd5f1c40eR210).